docs/SCD2-sparkcompute.md

```diff
@@ -17,6 +17,9 @@ int, long, float, double, bytes or string type.
 **End Date Field:** The name of the end date field. The sorted results are iterated to compute the value of this field based
 on the start date.
 
+**End Date Offset:** The offset to compute the end date. The end date will be computed as the next start date minus this offset.
+The unit is microseconds.
```

Comment:
microseconds is pretty user unfriendly. To specify a value for 1 second, would have to put in 1000000. One day would be even more unfriendly.

Does the UI have a widget that can be used to make this easier? If not, we should consider using the format supported by the TimeParser in hydrator-common to allow things like '1d', and '1s'.

Reply:
The time parser in hydrator common only supports up to milliseconds, it only supports time unit to seconds. It is not feasible for our cases.